app/views/contribute/canonical_data.erb

```diff
@@ -56,7 +56,7 @@
             </p>
 
             <p>
-              The process for extracting canonical data is described in the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">Contributing Guide</a>.
+              The process for extracting canonical data is described in the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">Contributing Guide</a>.
```

Comment:
This documentation now lives in https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/improving-consistency-across-tracks.md